core/checksums/pom.xml

```diff
@@ -51,13 +51,6 @@
             <version>${awsjavasdk.version}</version>
         </dependency>
 
-        <dependency>
-            <groupId>software.amazon.awssdk.crt</groupId>
-            <artifactId>aws-crt</artifactId>
-            <version>${awscrt.version}</version>
-            <optional>true</optional>
```

Comment:
Does this need to be removed? It's an optional dependency already so it wouldn't be loaded at runtime

Reply:
Yes ,Users needs to explicitly add the crt depencies if they want to load the CRT classes. 
We donot need it is as compile time as well as not as optional since we are loading classes at runtime,

I needed this dependency for testing so kept it as test dependency